packages/web-ui/VideoPlayer/index.tsx

```diff
@@ -4,10 +4,11 @@ import { useRouter } from 'next/router'
 import type { FC } from 'react'
 import React, { useCallback, useEffect, useState } from 'react'
 import { toast } from 'react-hot-toast'
-import { IPFS_GATEWAY, IS_MAINNET, LENSTUBE_WEBSITE_URL } from 'utils'
+import { IPFS_GATEWAY, IS_MAINNET, LENSTUBE_WEBSITE_URL, MUX_ENV } from 'utils'
 import useCopyToClipboard from 'utils/hooks/useCopyToClipboard'
 
 import SensitiveWarning from './SensitiveWarning'
+import mux from 'mux-embed'
```

Comment:
we can install the `mux-embed` lib in `web-ui` package. 🙇🏻 

In case required -`yarn workspace web-ui add mux-embed`

Reply:
Sure, that sounds good!

I didn't know there were two package :D

Reply:
We can only install it in `web-ui` package so that no need to install it on the web and embed apps separately. 🙇🏻